src/models/vae_lit_module.py

```diff
@@ -0,0 +1,32 @@
+from functools import partial
+from typing import Any
+
+import torch
+from lightning import LightningModule
+from torch import Tensor
+from torch.distributions import kl_divergence
+from torch.distributions.normal import Normal
+from torch.nn.functional import kl_div, mse_loss
+from torch.optim import Optimizer
+
+from .components.observation_encoder.vae import VAE
+
+
+class VAELitModule(LightningModule):
+    def __init__(self, vae_net: VAE, optimizer: partial[Optimizer]):
```

Comment:
kl損失に係数項をつけて調整可能にする必要があります。よくKL損失が強すぎて潜在空間が潰れてしまうことがよくあるので。
```suggestion
    def __init__(self, vae_net: VAE, optimizer: partial[Optimizer], kl_coef: float = 1.0):
```

```py
    def training_step(self, ...):
        loss = rec_loss + self.hparams.kl_coef * kl_loss
        return loss
```